File: NewsYuTechs/NewsyuTechs.BL/DTOs/Token/TokenDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NewsYuTechs.BL
{
    public class TokenDTO
    {
        public string? Token { get; set; }
        public DateTime ExpiryDate { get; set; }

        public string? Username { get; set; } 
        public string? User_id { get; set; }
    }
}

File: YuTechsAngular/src/app/_Models/TokenDTO.ts
export class TokenDTO {
    constructor(
        private _token: string,
        public userName: string,
        public _tokenExpirationDate: Date,
        public user_id:string,
    ) {}

    get token() {
        if (!this._tokenExpirationDate || new Date() > this._tokenExpirationDate) {
            return null;
        }
        return this._token;
    }
}